refactoring Istio dashboard and add control plane dashboard (#13590)
kyma-project/kyma: resources/istio-resources/files/dashboards/README.md

# Istio Dashboards

> Dashboards are form official Istio dashboards with minor changes: https://istio.io/latest/docs/ops/integrations/grafana/#configuration

Reviewer: ```suggestion
Istio Dashboards in Kyma are based on the official [Istio dashboards](https://istio.io/latest/docs/ops/integrations/grafana/#configuration) with minor modifications:
```

>  - removing DS_PROMETHEUS Grafana variable

Reviewer: ```suggestion
 - Removed `DS_PROMETHEUS` Grafana variable
```

>  - removing `__inputs` and `__requires` fields

Reviewer: ```suggestion
 - Removed **__inputs** and **__requires** fields
```

>  - adding tags: `"tags": ["service-mesh", "kyma"],`

Reviewer: ```suggestion
 - Added tags: **"tags": ["service-mesh", "kyma"],**
```
